quam_builder/architecture/superconducting/qubit_pair/fixed_frequency_transmon_pair.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, Any, Optional, Union
+from dataclasses import field
+from qm.qua import align, wait
+
+from quam.core import quam_dataclass
+from quam.components.quantum_components import QubitPair
+from quam_builder.architecture.superconducting.components.cross_resonance import (
+    CrossResonanceIQ,
+    CrossResonanceMW,
+)
+from quam_builder.architecture.superconducting.components.zz_drive import (
+    ZZDriveIQ,
+    ZZDriveMW,
+)
+
+
+__all__ = ["FixedFrequencyTransmonPair"]
+
+
+@quam_dataclass
+class FixedFrequencyTransmonPair(QubitPair):
```

Comment:
Now that we don't specify the type of the target and control qubit, how can we get type hinting?